src/panels/Menu/TopMenuBar/TopMenu.tsx

```diff
@@ -0,0 +1,39 @@
+import { PropsWithChildren } from 'react';
+import {
+  ArrowPosition,
+  FloatingAxesOffsets,
+  FloatingPosition,
+  Menu,
+  MenuProps
+} from '@mantine/core';
+
+interface Props extends MenuProps, PropsWithChildren {
+  menuItemTabIndex?: -1 | 0;
+  position?: FloatingPosition;
+  offset?: number | FloatingAxesOffsets;
+  withArrow?: boolean;
+  arrowPosition?: ArrowPosition;
+}
+
+export function TopMenu({
```

Comment:
I know we discussed this, but looking at it now this name seems too generic IMO. It sounds like the entire top menu, and not the individual menus... 

Don't have a good suggestion for another name though... Maybe `MenuWrapper` , `TopStyledMenu` ?   Or similar...?   Naming is so hard...